oceanbase-ce/README.md

```diff
@@ -1,47 +1,40 @@
+## About
+[oceanbase-ce](https://hub.docker.com/r/oceanbase/oceanbase-ce) is a test image for users to quickly setup an OceanBase environment. 
 
-**WARNING**
-
-- The `oceanbase-ce` docker image is just used for study or test;
-- Please use [ob-operator](https://github.com/oceanbase/ob-operator) instead if you want to deploy OceanBase on Kubernetes;
-- You should not deploy it with important data as it is not designed to use in production environment.
-
-Reasons:
-
-1. The cluster contains only one instance, so there is no disaster tolerant ability;
-2. It is very difficult to recover after failure because docker container cannot be started if the OceanBase instance started failed, which means you may lost the container and the data in it;
-3. K8s can not restart a new pod because the container still exists after the observer process quit.
+Before proceeding to the next step, please be aware of the following considerations:
+- This image is for test purpose, so don't use it for production;
+- This image only supports to setup single instance cluster;
+- This image is not to run on Kubernetes, if you have the need to run containerized OceanBase on Kubernetes, please checkout [ob-operator](https://github.com/oceanbase/ob-operator) for detail.
 
 ## Prerequisite
 
-Before you deploy `oceanbase-ce`, please do the following checks:
+Before you start to deploy `oceanbase-ce`, please make sure the following requirements are met:
 
-- Make sure that your machine has enough resource that can execute at least 2 physical cores and 8GB memory.
-- Your machine has installed the latest [Docker](https://docs.docker.com/get-docker/).
-- You have started the Docker service on your machine.
+- Make sure that your machine has enough resource that can provide least 2 physical cores and 8GB memory.
+- Your machine has installed and started [Docker](https://docs.docker.com/get-docker/).
 
 ## Start an OceanBase instance
 
-To start an OceanBase instance, run this command:
+To start an OceanBase instance, run the following command:
 
 ```bash
-# deploy mini instance
+# deploy mini mode instance
 docker run -p 2881:2881 --name oceanbase-ce -d oceanbase/oceanbase-ce
 
-# deploy an instance of the largest size according to the current container
+# deploy an instance to use the full resource of container
 docker run -p 2881:2881 --name oceanbase-ce -e MODE=normal -d oceanbase/oceanbase-ce
 
-# deploy a quick-start instance in any mode as desired to the current container
+# deploy an instance using fastboot mode
 docker run -p 2881:2881 --name oceanbase-ce -e MODE=slim -d oceanbase/oceanbase-ce
 
-# deploy an instance and execute init sqls
+# deploy an instance and execute init sqls after bootstrap
 docker run -p 2881:2881 --name oceanbase-ce -v {init_sql_folder_path}:/root/boot/init.d -d oceanbase/oceanbase-ce
```

Comment:
Add a note to remind users not to change passwords by init sql, but to use OB_TENANT_PASSWORD instead.

Reply:
OK